crates/weaver_semconv_gen/src/lib.rs

```diff
@@ -440,55 +390,4 @@ mod tests {
         ));
         Ok(())
     }
-
-    #[test]
-    fn test_http_semconv() -> Result<(), Error> {
```

Comment:
Do we still have unit tests for this crate?

Reply:
Parser has a ton of unit tests, and there's a minimal template-engine test which was the only template engine test we were using.

I don't think it's worth re-testing whether we resolve semconv or how JSON is formed, so it just makes sure that a markdown file is placed in markdown snippets.

Before we had to test all the details of the printlns.